auacm/app/static/js/controllers/JudgeController.js

```diff
@@ -29,10 +29,10 @@ app.controller('JudgeController', ['$scope', '$http', '$routeParams', '$window',
             submission.testNum = 0;
             $scope.submitted.push(submission);
         }, function(response) {
-                console.log("error");
+            console.log("error");
```

Comment:
Can we make this a bit more helpful? At least for admins, it would be nice to know _what_ went wrong. Just printing the response object would be enough.